views/partials/linking-data/firefox-monitor-product.hbs

```diff
@@ -0,0 +1,34 @@
+<script type="application/ld+json">
```

Comment:
I think this would require `unsafe-inline` in our CSP which we can't do. Is there an example of `application/ld+json` scripts hosted at a `src=` url?

Reply:
https://support.google.com/webmasters/thread/9589175?hl=en
https://webmasters.stackexchange.com/questions/104024/linking-external-json-ld-schema-from-schema-org

Seems to imply that it isn’t possible/supported.
First link is a couple months old. Second link is a couple years. Not sure if we could push a branch to Heroku and test, or do a merge/revert if it doesn’t work.